packages/core/package.json

```diff
@@ -11,7 +11,8 @@
     "test": "npm-run-all -s test:unit test:performance test:package",
     "test:watch": "karma start ./karma.conf.js --auto-watch=true --single-run=false --coverage=false",
     "test:package": "yarn run package-check --cwd ./dist/core",
-    "test:unit": "tsc --b ./tsconfig.spec.json --force && karma start ./karma.conf.js && yarn run build:package",
+    "test:unit": "tsc --b ./tsconfig.spec.json --force && karma start && karma start ./karma.conf.js && yarn run build:package",
+    "test:dev": "stmux -w always -e ERROR -m beep,system [ \"tsc --b ./tsconfig.spec.json -w --force\" : \"sleep 20 && yarn run build:watch\" : \"sleep 40 && karma start ./karma.conf.js --auto-watch=true --single-run=false --coverage=false\" ]",
```

Comment:
I added a thing so I could run everything in one window and see when it failed in one place rather than two. I wanted to watch src and spec and be able to debug things in the browser.